### ISSUE TITLE
[IntelliJ][ReportedByUser] Uncaught Exception  com.intellij.diagnostic.PluginException: 644 ms to call on EDT DeployFunctionAppAction#update@MainMenu (com.microsoft.azure.toolkit.intellij.legacy.function.action.DeployFunctionAppAction). Revise AnAction.getActionUpdateThread property [Plugin: com.microsoft.tooling.msservices.intellij.azure]

### ISSUE THREAD

**Issue by rezashamsca** [microsoft/azure-tools-for-java#7561]
IntelliJ build version: 2023.1.2 IC-231.9011.34
OS: Mac OS X
JDK: JetBrains s.r.o. 17.0.6
Plugin version: 3.76.1-2023.1
Additional Info: None
Parent component: 
```
com.intellij.openapi.wm.impl.IdeRootPane[,0,0,1611x967,invalid,layout=javax.swing.JRootPane$RootLayout,alignmentX=0.0,alignmentY=0.0,border=com.intellij.ui.ToolbarUtil$3@3c3f8acc,flags=449,maximumSize=,minimumSize=,preferredSize=]
```
Error message: 
```
IdeaLoggingEvent[message=null, throwable=com.intellij.diagnostic.PluginException: 644 ms to call on EDT DeployFunctionAppAction#update@MainMenu (com.microsoft.azure.toolkit.intellij.legacy.function.action.DeployFunctionAppAction). Revise AnAction.getActionUpdateThread property [Plugin: com.microsoft.tooling.msservices.intellij.azure]
	at com.intellij.diagnostic.PluginProblemReporterImpl.createPluginExceptionByClass(PluginProblemReporterImpl.java:23)
		at com.microsoft.azure.toolkit.intellij.legacy.function.runner.core.FunctionUtils.isFunctionProject(FunctionUtils.java:183)
		at com.microsoft.azure.toolkit.intellij.legacy.function.action.DeployFunctionAppAction.update(DeployFunctionAppAct

**Comment by wangmingliang-ms** [comment]
this issue should have been fixed in the latest v3.77.0. Feel free to reopen it if you have this problem again.

**Comment by wangmingliang-ms** [comment]
this issue should have been fixed in the latest v3.77.0. Feel free to reopen it if you encounter this problem again.
